Issue deploying to prod

@claude give me the deploy.yml in its entirety to paste since you don't have access to edit it. 

We failed to deploy the pages to prod with this error:

﻿2025-12-24T05:29:26.1481341Z ##[group]Run # Deploy to production environment
# Deploy to production environment
DEPLOY_OUTPUT=$(wrangler pages deploy public --project-name kris-kryngle-coordinator --branch main 2>&1 | tee /dev/stderr)
echo "DEPLOY_OUTPUT<<EOF" >> $GITHUB_ENV
echo "$DEPLOY_OUTPUT" >> $GITHUB_ENV
echo "EOF" >> $GITHUB_ENV

# Extract deployment URL (pages.dev URL)
DEPLOY_URL=$(echo "$DEPLOY_OUTPUT" | grep -oP 'https://[^\s]+\.pages\.dev' | head -1)

# Also try to extract custom domain URL
CUSTOM_URL=$(echo "$DEPLOY_OUTPUT" | grep -oP 'https://[^\s]+\.dilger\.dev' | head -1)

# Prefer custom domain if available, otherwise use pages.dev
if [ -n "$CUSTOM_URL" ]; then
  DEPLOY_URL="$CUSTOM_URL"
elif [ -z "$DEPLOY_URL" ]; then
  # Fallback: try to find any HTTPS URL
  DEPLOY_URL=$(echo "$DEPLOY_OUTPUT" | grep -oP 'https://[^\s]+' | head -1)
fi

echo "DEPLOY_URL=$DEPLOY_URL" >> $GITHUB_ENV
echo "CUSTOM_URL=$CUSTOM_URL" >> $GITHUB_ENV
echo "✅ Deployed to: $DEPLOY_URL"
shell: /usr/bin/bash -e {0}
env:
  CLOUDFLARE_API_TOKEN: ***
  CLOUDFLARE_ACCOUNT_ID: ***

 ⛅️ wrangler 4.56.0
───────────────────
▲ [WARNING] Warning: Your working directory is a git repo and has uncommitted changes

  To silence this warning, pass in --commit-dirty=true



✘ [ERROR] Running configuration file validation for Pages:

    - Configuration file for Pages projects does not support "routes"

It worked great for the PR

@claude updated that and got this:

﻿2025-12-24T05:36:15.4235356Z ##[group]Run # Deploy to production environment
# Deploy to production environment
DEPLOY_OUTPUT=$(wrangler pages deploy public --project-name kris-kryngle-coordinator --branch main 2>&1 | tee /dev/stderr)
echo "DEPLOY_OUTPUT<<EOF" >> $GITHUB_ENV
echo "$DEPLOY_OUTPUT" >> $GITHUB_ENV
echo "EOF" >> $GITHUB_ENV

# Extract deployment URL (pages.dev URL)
DEPLOY_URL=$(echo "$DEPLOY_OUTPUT" | grep -oP 'https://[^\s]+\.pages\.dev' | head -1)

# Also try to extract custom domain URL
CUSTOM_URL=$(echo "$DEPLOY_OUTPUT" | grep -oP 'https://[^\s]+\.dilger\.dev' | head -1)

# Prefer custom domain if available, otherwise use pages.dev
if [ -n "$CUSTOM_URL" ]; then
  DEPLOY_URL="$CUSTOM_URL"
elif [ -z "$DEPLOY_URL" ]; then
  # Fallback: try to find any HTTPS URL
  DEPLOY_URL=$(echo "$DEPLOY_OUTPUT" | grep -oP 'https://[^\s]+' | head -1)
fi

echo "DEPLOY_URL=$DEPLOY_URL" >> $GITHUB_ENV
echo "CUSTOM_URL=$CUSTOM_URL" >> $GITHUB_ENV
echo "✅ Deployed to: $DEPLOY_URL"
shell: /usr/bin/bash -e {0}
env:
  CLOUDFLARE_API_TOKEN: ***
  CLOUDFLARE_ACCOUNT_ID: ***

 ⛅️ wrangler 4.56.0
───────────────────
▲ [WARNING] Warning: Your working directory is a git repo and has uncommitted changes

  To silence this warning, pass in --commit-dirty=true



✘ [ERROR] Running configuration file validation for Pages:

    - Configuration file for Pages projects does not support "routes"


🪵  Logs were written to "/home/runner/.config/.wrangler/logs/wrangler-2025-12-24_05-36-15_880.log"
✅ Deployed to:

--- a/wrangler.toml
+++ b/wrangler.toml
@@ -4,14 +4,6 @@ compatibility_date = "2024-12-20"
 # Pages configuration
 pages_build_output_dir = "public"
 
-# Production deployment
-[env.production]
-name = "kris-kryngle-coordinator"
-vars = { ENVIRONMENT = "production" }
-routes = [
-  { pattern = "kris-kryngle-coordinator.dilger.dev/*", zone_name = "dilger.dev" }
-]
-
 # Uncomment to use KV namespaces
 # [[kv_namespaces]]
 # binding = "MY_KV"

